spec/requests/prescriptions_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to test the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator. If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails. There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+
+RSpec.describe '/prescriptions', type: :request do
+  # This should return the minimal set of attributes required to create a valid
+  # Prescription. As you add validations to Prescription, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) do
+    {
+      product: 'Paracetamol', quantity: 10, symbol: '*', dose: 2, days: 3, stock: 100
+    }
+  end
+
+  let(:invalid_attributes) do
+    {
+      product: '', quantity: '', symbol: '', dose: '', days: '', stock: ''
+    }
+  end
+
+  # This should return the minimal set of values that should be in the headers
+  # in order to pass any filters (e.g. authentication) defined in
+  # PrescriptionsController, or in your router and rack
+  # middleware. Be sure to keep this updated too.
+  let(:valid_headers) do
+    { 'Content-Type' => 'application/json' }
+  end
+
+  describe 'GET /index' do
+    it 'renders a successful response' do
+      Prescription.create! valid_attributes
```

Comment:
- [ ] You have an error message originating from this line
![image](https://user-images.githubusercontent.com/45460192/219386741-985cfa9e-3b9f-4899-a268-ba6a59be80aa.png)